config/v1/types_apiserver.go

```diff
@@ -52,6 +52,7 @@ type APIServerSpec struct {
 	// server from JavaScript applications.
 	// The values are regular expressions that correspond to the Golang regular expression language.
 	// +optional
+	// +listType=atomic
```

Comment:
These `+listType=atomic` are required to make the `verify-crd-schema` pass, which fails the test saying lack of SSA tags in the generated CRD. Alternative to adding these is to override the specific test but this retains the API violations as-is today.